src/components/App/App.tsx

```diff
@@ -12,23 +14,44 @@ const App: FC = () => {
       testSet: '',
     },
     {
-      name: 'SMART Bulk Tests',
-      fhirServer:
-        'https://bulk-data.smarthealthit.org/eyJlcnIiOiIiLCJwYWdlIjoxMDAwLCJkdXIiOjEwLCJ0bHQiOjE1LCJtIjoxLCJzdHUiOjR9/fhir',
-      testSet: 'Bulk data tests (via BDT)',
-    },
-    {
-      name: 'US Core v3.1.1 with Inferno Reference Server',
+      name: 'Demonstration Sequence',
```

Comment:
My guess is that eventually we'll want these presets to be pulled from the API as well, but for now this works.

Reply:
Yeah, there's no server support for presets yet, but when there is I'll pull it at the same place I pull the test suite info.